# Raw data from a test sonar sensor

float64 timestamp
float64[10] detections  # Distance of detections
float64[10] degrees     # Angle of detections
